extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerAuthorizer.java

```diff
@@ -43,6 +46,11 @@
 
     @Override
     public CheckResult apply(RoutingContext routingContext, SecurityIdentity identity) {
+        if (delegate == null) {
+            LOG.debug(
+                    "Keycloak Policy Enforcer has not been initialized - please make sure 'quarkus.oidc.enabled' is not set to 'false'");
+            throw new AuthenticationFailedException();
+        }
```

Comment:
Can we also explain why the Authentication failed to the caller here?

Reply:
I was thinking, since this code is executed as part of the user authentication process, it should be 401. In fact first I coded an `IllegalStateException` but then I changed my mind, but now that you asked, I wonder if returning 500 would be best as it is really a server configuration issue. What would you prefer yourself ?

Reply:
Returning 500 makes more sense to me

Reply:
@gastaldi thanks, I'll fix during the next few of days, have to sign off now, thanks for starting the review, enjoy the weekend :-)

Reply:
Likewise, have fun! 😉